changelog/generated/v1.5.0.md

```diff
@@ -0,0 +1,37 @@
+## v1.5.0
+
+### Additions
+
+- Add support for markers for files with the `yml` extension. More info: [#kubernetes-sigs/kubebuilder#1907](https://github.com/kubernetes-sigs/kubebuilder/pull/1907). ([#4402](https://github.com/operator-framework/operator-sdk/pull/4402))
+- (go/v3) Added the `--force` option to the `create webhook` command. More info: [#kubernetes-sigs/kubebuilder#1903](https://github.com/kubernetes-sigs/kubebuilder/pull/1903). ([#4402](https://github.com/operator-framework/operator-sdk/pull/4402))
+- For Go-based operators, add `ErrorIfCRDPathMissing` config by default to the `suite_tests.go`. More info [#kubernetes-sigs/kubebuilder#1910](https://github.com/kubernetes-sigs/kubebuilder/pull/1910). ([#4402](https://github.com/operator-framework/operator-sdk/pull/4402))
+- For Ansible-based operators, allow passing a --graceful-shutdown-timeout to configure the duration the manager should wait before stopping. ([#4571](https://github.com/operator-framework/operator-sdk/pull/4571))
+- (go/v3) Create and bind to a non-default service account ([kubebuilder#2070](https://github.com/kubernetes-sigs/kubebuilder/pull/2070)). ([#4626](https://github.com/operator-framework/operator-sdk/pull/4626))
+
+### Changes
+
+- **Breaking change**: For Go-based operators, migrate your project to use the project version stable. ([#4402](https://github.com/operator-framework/operator-sdk/pull/4402))
+- **Breaking change**: For Ansible/Helm-based operators, usage of project config stable version. ([#4402](https://github.com/operator-framework/operator-sdk/pull/4402))
+- For Go-based operators using go/v3 plugin only, upgrade the sigs.k8s.io/kubebuilder-declarative-pattern dependency (used only to gen api with --pattern=addon). More info: [#kubernetes-sigs/kubebuilder#1946](https://github.com/kubernetes-sigs/kubebuilder/pull/1946). ([#4402](https://github.com/operator-framework/operator-sdk/pull/4402))
+- For Go-based operators, add Makefile help. ([#4402](https://github.com/operator-framework/operator-sdk/pull/4402))
+- For Go-based operators, Makefile scaffold was improved. ([#4402](https://github.com/operator-framework/operator-sdk/pull/4402))
```

Comment:
This is kinda ambiguous. Nothing about the Makefile was changed except for adding help and some reorganization, all cosmetic.
```suggestion
```